aws/resource_aws_db_cluster_snapshot.go

```diff
@@ -167,6 +170,24 @@ func resourceAwsDbClusterSnapshotRead(d *schema.ResourceData, meta interface{})
 	d.Set("storage_encrypted", snapshot.StorageEncrypted)
 	d.Set("vpc_id", snapshot.VpcId)
 
+	if err := saveTagsRDS(conn, d, aws.StringValue(snapshot.DBClusterSnapshotArn)); err != nil {
+		log.Printf("[WARN] Failed to save tags for RDS DB Cluster Snapshot (%s): %s", d.Id(), err)
```

Comment:
Nit: In the future we will be requiring these to return an error instead of just logging 👍